packages/chat/src/components/VoiceWidget/ShrinkingCircle.component.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useRef, useState } from 'react';
+
+import { shrinkingCircle } from './ShrinkingCircle.css';
+
+export const ShrinkingCircle = () => {
+  const [radius, setRadius] = useState(54);
+  const audioContextRef = useRef<AudioContext | null>(null);
+  const analyserRef = useRef<AnalyserNode | null>(null);
+
+  useEffect(() => {
+    const initAudio = async () => {
+      if (audioContextRef.current) return;
```

Comment:
maybe this should be a dependency of the `effect` here if it needs to be triggered when this is changed (and not triggered again after) ?